### PR TITLE
docs: change logo to X

### PR DESCRIPTION
**Pull request by validcube**
Goodbye Twitter Bird...

Not sure why the url is difference from others

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
Is this meant to be a draft?

**Comment by validcube** [comment]
This PR is draft 

see: 
* https://github.com/orgs/community/discussions/61110 
* https://github.com/github/cmark-gfm/issues/342

**Comment by validcube** [comment]
done:tm:! @oSumAtrIX

**Comment by oSumAtrIX** [comment]
This has to be changed in the branding repo too

**Comment by validcube** [comment]
> This has to be changed in the branding repo too

Done:tm:
- https://github.com/ReVanced/revanced-branding/pull/19

File: profile/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
        <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032213-faf25ab8-0bc3-4a94-a730-b524c96df124.png" />
    </a>&nbsp;&nbsp;&nbsp;
    <a href="https://twitter.com/revancedapp">
-       <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032018-6da37214-7474-4641-a1da-7af7db3a31cd.png" />
+       <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
```
Comment (oSumAtrIX) [inline profile/README.md:32]: Please add a light and dark-themed icon. On a dark background, this logo is invisible.

**Pull request merged**